src/gui/mainwindow.cpp

```diff
@@ -2011,9 +2011,9 @@ void MainWindow::installPython()
     setCursor(QCursor(Qt::WaitCursor));
     // Download python
 #ifdef QBT_APP_64BIT
-    const QString installerURL = "https://www.python.org/ftp/python/3.8.1/python-3.8.1-amd64.exe";
+    const QString installerURL = "https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe";
```

Comment:
Have you tested this version? My experience is that sometimes the releases are not so stable, at least I saw `pip` not working in some version (and obviously the user would expect to use this installed python for other things).

Reply:
I didn't test but should be fine. We are using the installed version, this is only if the user doesn't have any version installed. In that case this version will be used only by us.